resources/lib/dialogs/DialogMovieInfo.py

```diff
@@ -209,13 +209,13 @@ def rating_button(self, control_id):
 
     @ch.click(ID_BUTTON_PLAY_RESUME)
     def play_noresume_button(self, control_id):
-        self.exit_script()
+        self.exit()
         xbmc.executebuiltin("Dialog.Close(movieinformation)")
         kodijson.play_media("movie", self.info["dbid"], True)
 
     @ch.click(ID_BUTTON_PLAY_NORESUME)
     def play_resume_button(self, control_id):
-        self.exit_script()
```

Comment:
Wouldn't another solution for this be simply adding "control_id";
`self.exit_script(control_id)`